Simperium/UIViewController+Simperium.m

```diff
@@ -0,0 +1,32 @@
+//
+//  UIViewController.m
+//  Simperium
+//
+//  Created by Jorge Leandro Perez on 10/27/15.
+//  Copyright © 2015 Simperium. All rights reserved.
+//
+
+#import "UIViewController+Simperium.h"
+
+@implementation UIViewController (Simperium)
+
+- (BOOL)sp_isViewOnscreen
+{
+    BOOL visibleAsRoot          = self.view.window.rootViewController == self;
+    BOOL visibleAsTopOnStack    = self.navigationController.topViewController == self;
+    BOOL visibleAsPresented     = [self.view.window.rootViewController sp_leafViewController] == self;
+    
+    return visibleAsRoot || visibleAsTopOnStack || visibleAsPresented;
```

Comment:
Quick question, is the goal just to know if the view is on screen or is there more being tested? If its just to know if the view is on screen checking wouldn't `self.view.window != nil` do the trick?